opengamedata/opengamedata-website
# FD-20 data template

Testing instructions:
To get data from API, opengamedata-website-api needs to be running on localhost:5000

Load a game by appending `?game=GAMENAME` (e.g., LAKELAND) to `/gamedata.php`
- Game should be present in `data/game_list.json`
- If testing with LAKELAND and current main branch in API, you need to pass a month and year with data in `file_list.json` (since most recent month/year is hard-coded in api).
Add a year and month to `gamedata.php` line 29: `$response_obj = services\getGameUsage($game_id,2021,8);`
